lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -1258,11 +1258,11 @@
     }
 
     .three-column-layout {
-       max-width: 33%;
+       max-width: 30%;
```

Comment:
The issue was padding-left and padding-right are both 1%. Therefore the columns would end up taking > 100% on initial render (not sure why it wasn't an issue after dynamically adding and removing the layout classes).

I have reduced these max-widths so the padding values will fit.